pkg/kubectl/cmd/run.go

```diff
@@ -85,25 +86,62 @@ var (
 )
 
 type RunObject struct {
```

Comment:
Like I said before split this part to a separate PR, I'm not even looking at it atm.